EIPS/eip-3238.md

```diff
@@ -0,0 +1,42 @@
+---
+eip: 3238
+title: Vatnajoekull Difficulty Bomb Delay
```

Comment:
Hard fork coordination happens separate from the EIP process in https://github.com/ethereum/eth1.0-specs/.  An EIP is just a technical document that describes a change, and has no concept of Ethereum mainnet fork names.

```suggestion
title: Difficulty Bomb Delay to Summer 2020
```

Reply:
OK but fixed the year ;-)

```suggestion
title: Difficulty Bomb Delay to Summer 2022
```